pom.xml

```diff
@@ -49,14 +48,12 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>structs</artifactId>
-            <version>1.2</version>
+            <version>324.va_f5d6774f3a_d</version>
```

Comment:
Might be good to introduce the bom so that a specific version doesn't need to be here.

Reply:
Added BOM.